airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java

```diff
@@ -503,7 +503,7 @@ String dedupFinalTable(final StreamId id,
               `_airbyte_raw_id` IN (
                 SELECT `_airbyte_raw_id` FROM (
                   SELECT `_airbyte_raw_id`, row_number() OVER (
-                    PARTITION BY ${pk_list} ORDER BY ${cursor_name} DESC, `_airbyte_extracted_at` DESC
+                    PARTITION BY ${pk_list} ORDER BY ${cursor_name} DESC NULLS LAST, `_airbyte_extracted_at` DESC
```

Comment:
* [Snowflake](https://docs.snowflake.com/en/sql-reference/constructs/order-by) can do this! 
* [Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_ORDER_BY_clause.html) can do this!
* [Postgres](https://www.postgresql.org/docs/current/queries-order.html#:~:text=The%20NULLS%20FIRST%20and%20NULLS,order%2C%20and%20NULLS%20LAST%20otherwise.) can do this!
* [Clickhouse](https://clickhouse.com/docs/en/sql-reference/statements/select/order-by) can do this!
* [MySQL](https://www.designcise.com/web/tutorial/how-to-order-null-values-first-or-last-in-mysql) can do this (in a very odd way)!
* [Databrikcs](https://docs.databricks.com/sql/language-manual/sql-ref-syntax-qry-select-orderby.html) can do this! 

Seems like this is a thing for enough databases to roll with!